libpod/container_internal_linux.go

```diff
@@ -482,6 +482,21 @@ func (c *Container) checkpoint(ctx context.Context, options ContainerCheckpointO
 	if c.state.State != ContainerStateRunning {
 		return errors.Wrapf(ErrCtrStateInvalid, "%q is not running, cannot checkpoint", c.state.State)
 	}
+
+	// Create the CRIU log file and label it
+	logFile, err := os.Create(filepath.Join(c.bundlePath(), "dump.log"))
+	if err != nil {
+		return errors.Wrapf(err, "failed to create CRIU log file %q", filepath.Join(c.bundlePath(), "dump.log"))
+	}
+	logFile.Close()
+	if os.Chmod(filepath.Join(c.bundlePath(), "dump.log"), 0600); err != nil {
+		return errors.Wrapf(err, "failed to chmod CRIU log file %q", filepath.Join(c.bundlePath(), "dump.log"))
+	}
+	err = label.SetFileLabel(filepath.Join(c.bundlePath(), "dump.log"), c.MountLabel())
```

Comment:
`if err := label.SetFileLabel(...); err != nil {` - combines this with the next line